service/src/integrationTest/kotlin/fi/espoo/evaka/invoicing/data/FeeDecisionSearchTest.kt

```diff
@@ -212,6 +214,50 @@ class FeeDecisionSearchTest : PureJdbiTest(resetDbBeforeEach = true) {
         }
     }
 
+    @Test
+    fun `starting children filter search`() {
+        db.transaction { tx ->
+            tx.upsertFeeDecisions(
+                listOf(
+                    decisionFixture(
+                        headOfFamily = testAdult_3.id,
+                        children =
+                            listOf(childFixture(testChild_3, testDaycare.id, serviceNeed = null))
+                    )
+                )
+            )
+        }
+        assertEquals(
+            1,
+            search(distinctiveParams = listOf(DistinctiveParams.NO_STARTING_PLACEMENTS)).size
+        )
+
+        assertEquals(
+            1,
+            search(
+                    areas = listOf(testArea.shortName),
+                    distinctiveParams = listOf(DistinctiveParams.NO_STARTING_PLACEMENTS)
+                )
+                .size
+        )
+
+        db.transaction { tx ->
+            tx.insertTestPlacement(
+                DevPlacement(
+                    childId = testChild_3.id,
+                    unitId = testDaycare.id,
+                    startDate = LocalDate.now(),
```

Comment:
Suggestion: don't use `LocalDate.now()`, because it adds an unnecessary random element to the test (we don't put random values to all String/Int/Long/Boolean fields either). If the *actual* date value is not important, prefer a constant `val today = LocalDate.of(something)` and use that. If we truly need "today's date" and the test depends on the time it's executed, refactor the underlying code to use mock clock instead. This applies to all LocalDate.now() calls that have been added.